lib/metamocha.js

```diff
@@ -0,0 +1,127 @@
+var Mocha = require('mocha'),
+    Context = require('mocha').Context,
+    metadataUi = require('./metadata_ui'), // eslint-disable-line no-unused-vars
```

Comment:
Can you explain what's happening with this line? Why are you suppressing the eslint warning?